lib/utils.pm

```diff
@@ -1221,4 +1223,25 @@ sub script_run_interactive {
     }
 }
 
+sub setup_autofs_server {
```

Comment:
not sure if utils.pm is right place, @asmorodskyi  what do you think about to place it to network_utils ?

Reply:
+1 let's move it to network_utils

Reply:
What about having a base class specific to autofs?

Reply:
Yes, a new class for autofs has been created in `lib/autofs_utils.pm` it contains `setup_autofs_server` and `check_autofs_service` functions that were before located in `lib/network_utils.pm`